Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:4.0
+// swift-tools-version:4.2
```

Comment:
I think this has some consequence on swift-pm rebuilding from scratch every time if the tools on CI don't match this version.

Reply:
We should create a new issue to discuss the effect this might currently be having on the `master` branch (as, ya know, `release/0.34.0` won't be active after we finish our work) 👍